packages/block/src/header-from-rpc.ts

```diff
@@ -28,6 +28,14 @@ export default function blockHeaderFromRpc(blockParams: any, options?: BlockOpti
     nonce,
   } = blockParams
 
+  let baseFeePerGas
+
+  // Check if the field baseFeePerGas is present in the block.
+  // This field was introduced after: https://eips.ethereum.org/EIPS/eip-1559
+  if (Object.prototype.hasOwnProperty.call(blockParams, 'baseFeePerGas')) {
```

Comment:
Thanks for the update. Sorry, don't want to be too picky, but can we do a simple `'baseFeePerGas' in blockParams` check?

Reply:
(apart from that we can really do a quick release here, we have releases planned for - likely - early next week, see #1327, and it should still be possible to get this in here)